src/evolutions/src/main/scala/play/api/db/slick/evolutions/internal/DBApiAdapter.scala

```diff
@@ -53,14 +53,21 @@ private[evolutions] object DBApiAdapter {
       val session = dbConfig.db.createSession()
       session.conn
     }
-    def getConnection(autocommit: Boolean): Connection = getConnection() // FIXME: auto-commit is ignored (in slick, I believe auto-commit is on by default, but need to double check)
+    def getConnection(autocommit: Boolean): Connection = {
+      val conn = getConnection()
+      conn.setAutoCommit(autocommit)
+      conn
+    }
     def withConnection[A](block: Connection => A): A = {
       dbConfig.db.withSession { session =>
         val conn = session.conn
         block(conn)
       }
     }
-    def withConnection[A](autocommit: Boolean)(block: Connection => A): A = withConnection(block) // FIXME: auto-commit is ignored (in slick, I believe auto-commit is on by default, but need to double check)
+    def withConnection[A](autocommit: Boolean)(block: Connection => A): A = withConnection { conn =>
+      conn.setAutoCommit(true)
```

Comment:
The `autocommit` parameter should be honored and not forced to `true`, so :
`conn.setAutoCommit(autocommit)`